integration/integration.go

```diff
@@ -57,6 +57,18 @@ func GetIntegrationName(result string) string {
 	return res.Data[0].Name
 }
 
+func GetContainerRegistryIntegration(result string) api.ContainerRegIntegration {
```

Comment:
q: why are we doing this? I'm concerned about us splitting a string and the string itself changing. This feels a little brittle.

Reply:
It certainly could be improved. The string being split is the terraform output, which is consistent. If there was no output or no id in the output the tf step would have failed. But I can look at pattern matching instead.